### ISSUE TITLE
When was train_ocr.py used?

### ISSUE THREAD

**Issue by mattroos** [MichalBusta/E2E-MLT#11]
@MichalBusta, congrats on the great performance by your newest model.  Can you give a few more details on your training?  When/why did you use the train_ocr.py code?

**Comment by Linchenguang** [comment]
Hello, mattroos, I used train_ocr.py to pretrain the ocr net, when the output looks good, I used the saved model to restore the network in train.py and begin trainning. It works fine! May be it is for pretraining ocr and saving training time.

**Comment by MichalBusta** [comment]
Hi Matt,

On 10/12/2018 23:06, mattroos wrote:
>
> @MichalBusta <https://github.com/MichalBusta>, congrats on the great 
> performance by your newest model. Can you give a few more details on 
> your training? When/why did you use the train_ocr.py code?
>
the train_ocr is mainly for experiments with stand-alone ocr. The ocr is 
fast to train, overall accuracy is more dependent on ocr model.

it is also good tool for application - you can train detector and just 
fine-tune on ocr data, since it is easy(ier) to generate crops with good 
distribution.

**Comment by mattroos** [comment]
Thanks, @MichalBusta and @Linchenguang!
